docs/how-to-use-customized-configuration.md

```diff
@@ -1,294 +0,0 @@
-# How to use a customized properties file with Scalar Helm Charts
-
-You can set your customized properties files with Scalar Helm Charts (ScalarDB, ScalarDL Ledger, ScalarDL Auditor, and ScalarDL Schema Loader). Also, you can specify some flags in addition to the customized properties file with the ScalarDL Schema Loader chart. This document explains how to set properties files and flags with Scalar Helm Charts.
-
-## Set your properties file to a custom values file
-
-1. Set your properties to the following keys in the custom values file.
-   * Keys
-     * `scalardb.databaseProperties` (ScalarDB)
-     * `ledger.ledgerProperties` (ScalarDL Ledger)
-     * `auditor.auditorProperties` (ScalarDL Auditor)
-     * `schemaLoading.databaseProperties` (ScalarDL Schema Loader)
-
-   * Example
-       * ScalarDB
-         ```yaml
-         scalardb:
-           databaseProperties: |
-             scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
-             scalar.db.username=postgres
-             scalar.db.password=postgres
-             scalar.db.storage=jdbc
-         ```
-       * ScalarDL Ledger
-         ```yaml
-         ledger:
-           ledgerProperties: |
-             scalar.db.contact_points=jdbc:postgresql://postgresql-ledger.default.svc.cluster.local:5432/postgres
-             scalar.db.username=postgres
-             scalar.db.password=postgres
-             scalar.db.storage=jdbc
-             scalar.dl.ledger.proof.enabled=true
-             scalar.dl.ledger.auditor.enabled=true
-             scalar.dl.ledger.proof.private_key_path=/keys/private-key
-         ```
-       * ScalarDL Auditor
-         ```yaml
-         auditor:
-           auditorProperties: |
-             scalar.db.contact_points=jdbc:postgresql://postgresql-auditor.default.svc.cluster.local:5432/postgres
-             scalar.db.username=postgres
-             scalar.db.password=postgres
-             scalar.db.storage=jdbc
-             scalar.dl.auditor.ledger.host=scalardl-ledger-envoy
-             scalar.dl.auditor.cert_path=/keys/certificate
-             scalar.dl.auditor.private_key_path=/keys/private-key
-         ```
-       * ScalarDL Schema Loader
-         ```yaml
-         schemaLoading:
-           databaseProperties: |
-             scalar.db.contact_points=jdbc:postgresql://postgresql-ledger.default.svc.cluster.local:5432/postgres
-             scalar.db.username=postgres
-             scalar.db.password=postgres
-             scalar.db.storage=jdbc
-         ```
-
-1. Deploy Scalar products with the above custom values file.  
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Use `Secret` resources to pass the credentials as the environment variables into the properties file
```

Comment:
I moved this section to [How to use Secret resources to pass the credentials as the environment variables into the properties file](https://github.com/scalar-labs/helm-charts/blob/main/docs/use-secret-for-credentilas.md).